packages/voice-compass/typedoc.cjs

```diff
@@ -0,0 +1,12 @@
+/** @type {import('typedoc').TypeDocOptions} */
+module.exports = {
+  sort: ["source-order", "kind", "instance-first", "alphabetical"],
+  entryPoints: ["./src/index.ts"],
+  out: "docs",
+  validation: { notExported: true, invalidLink: true, notDocumented: true },
+  treatWarningsAsErrors: true,
+  treatValidationWarningsAsErrors: true,
+  categoryOrder: ["Setup", "Client"],
+  plugin: ["typedoc-plugin-markdown"],
```

Comment:
the markdown plugin is a proxy for a minimally-styled version of typedoc's output.

for more examples of how typedoc usually looks in the wild:

https://typedoc.org/api/
https://typedoc.org/example/

(or with a non-standard theme:)
https://v1.baklava.tech/api/index.html#/@baklavajs%2Fcore